RELEASING.md

```diff
@@ -41,3 +69,38 @@ GITHUB_AUTH=xxxxx lerna-changelog --from=v1.0.0 --to=v2.0.0
 #### Update Changelog file
 
 From what `lerna-changelog` has generated, starts new Unreleased label. Follow the example set by recent Released label.
+
+On [GitHub Releases](https://github.com/open-telemetry/opentelemetry-js/releases), follow the example set by recent releases to populate a summary of changes, as well as a list of commits that were applied since the last release. Save it as a draft, don’t publish it. Don’t forget the tag -- call it `vx.y.z` and leave it pointing at `master` for now (this can be changed as long as the GitHub release isn’t published).
+
+## Create a new PR
+
+Create a pull request titled `chore: x.y.z release proposal`. The commit body should just be a link to the draft notes. Someone who can access draft notes should approve it, looking in particular for test passing, and whether the draft notes are satisfactory.
+
+## Merge and pull
+
+Merge the PR, and pull the changes locally (using the commands in the first step). Ensure that `chore: x.y.z release proposal` is the most recent commit.
+
+## Publish all packages
```

Comment:
Does this need to be a manual process? Why can we not use lerna publish for this?

Reply:
So instead of `npm publish` on all the packages, just use `lerna publish`?

Reply:
> So instead of npm publish on all the packages, just use lerna publish?

I believe that `lerna publish from-package` should do the same at that point.